apps/api/src/domain/models/audio-item/commands/translate-audio-item-name/translate-audio-item.command.integration.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { DTO } from '../../../../../types/DTO';
+import { assertCommandSuccess } from '../../../__tests__/command-helpers/assert-command-success';
+import { CommandAssertionDependencies } from '../../../__tests__/command-helpers/types/CommandAssertionDependencies';
+import { dummySystemUserId } from '../../../__tests__/utilities/dummySystemUserId';
+import { AudioItem } from '../../entities/audio-item.entity';
+import { TranslateAudioItemName } from './translate-audio-item-name.command';
+
+const commandType = 'TRANSLATE_AUDIO_ITEM_NAME';
+
+const existingAudioItem = getValidAggregateInstanceForTest(AggregateType.audioItem).clone({
+    name: new MultilingualText({
+        items: [
+            new MultilingualTextItem({
+                text: 'original name of the audio item',
+                role: MultilingualTextItemRole.original,
+                languageCode: LanguageCode.Chilcotin,
+            }),
+        ],
+    }),
+});
+
+const englishName = 'audio item name translated to English';
+
+const validPayload: TranslateAudioItemName = {
+    aggregateCompositeIdentifier: {
+        type: AggregateType.audioItem,
+        id: existingAudioItem.id,
+    },
+    languageCode: LanguageCode.English,
+    text: englishName,
+};
+
+const validCommandFSA = {
+    type: commandType,
+    payload: validPayload,
+};
+
+const buildValidCommandFSA = (): FluxStandardAction<DTO<TranslateAudioItemName>> => validCommandFSA;
+
+describe(commandType, () => {
+    let app: INestApplication;
+
+    let testRepositoryProvider: TestRepositoryProvider;
+
+    let commandHandlerService: CommandHandlerService;
+
+    let idManager: IIdManager;
+
+    let commandAssertionDependencies: CommandAssertionDependencies;
+
+    beforeAll(async () => {
+        ({ testRepositoryProvider, commandHandlerService, idManager, app } =
+            await setUpIntegrationTest({
+                ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+            }).catch((error) => {
+                throw error;
+            }));
+
+        commandAssertionDependencies = {
+            testRepositoryProvider,
+            idManager,
+            commandHandlerService,
+        };
+    });
+
+    afterAll(async () => {
+        await app.close();
+    });
+
+    beforeEach(async () => {
+        await testRepositoryProvider.testSetup();
+    });
+
+    describe('when the command is valid', () => {
+        it('should succeed', async () => {
+            await assertCommandSuccess(commandAssertionDependencies, {
+                systemUserId: dummySystemUserId,
+                buildValidCommandFSA,
+                initialState: new DeluxeInMemoryStore({
+                    [AggregateType.audioItem]: [existingAudioItem],
+                }).fetchFullSnapshotInLegacyFormat(),
+                checkStateOnSuccess: async ({
+                    aggregateCompositeIdentifier: { id: audioItemListId },
```

Comment:
Alias `id` as `audioItemId` not ~~audioItemListId~~.

Reply:
done